t/api/04-jobs.t

```diff
@@ -401,6 +401,18 @@ $t->json_is('/job/settings/DISTRI'  => undef);
 
 $post = $t->post_ok('/api/v1/jobs', form => {})->status_is(400);
 
+subtest 'job details' => sub {
+
+    $t->get_ok('/api/v1/jobs/99926')->status_is(200);
+    $t->json_is('/job/testresults' => undef, 'Test details are not present');
+
+    $t->get_ok('/api/v1/jobs/99926?details=1')->status_is(200);
```

Comment:
I don't like that this is a param. This should be a documented route as in jobs/99926/details